test/e2e-tests.sh

```diff
@@ -27,10 +27,90 @@
 
 source $(dirname $0)/../vendor/github.com/knative/test-infra/scripts/e2e-tests.sh
 
+readonly KNATIVE_EVENTING_RELEASE=https://knative-releases.storage.googleapis.com/eventing/latest/release.yaml
+
+# Names of the Resources used in the tests.
+readonly E2E_TEST_NAMESPACE=e2etest
+readonly E2E_TEST_FUNCTION_NAMESPACE=e2etestfn3
+
+# Helper functions.
+
+# Install the latest stable Knative/eventing in the current cluster.
+function start_latest_eventing() {
+  header "Starting Knative Eventing"
+  subheader "Installing Knative Eventing"
+  kubectl apply -f ${KNATIVE_EVENTING_RELEASE} || return 1
+  wait_until_pods_running knative-eventing || return 1
+}
+
+
+function teardown() {
+  teardown_events_test_resources
+  ko delete --ignore-not-found=true -f config/default.yaml
+  ko delete --ignore-not-found=true -f ${KNATIVE_EVENTING_RELEASE}
+
+  wait_until_object_does_not_exist namespaces knative-sources
+  wait_until_object_does_not_exist namespaces knative-eventing
+
+  wait_until_object_does_not_exist customresourcedefinitions containersources.sources.knative.dev
+  wait_until_object_does_not_exist customresourcedefinitions gcppubsubsources.sources.knative.dev
```

Comment:
This isn't part of the standard `default.yaml`.

Reply:
Good catch, fixed.